Formula/e/envelope.rb

```diff
@@ -0,0 +1,33 @@
+class Envelope < Formula
+  desc "Environment variables CLI tool"
+  homepage "https://github.com/mattrighetti/envelope"
+  url "https://github.com/mattrighetti/envelope/archive/refs/tags/0.3.11.tar.gz"
+  sha256 "1a378564b07e041fbf3212655e8c6442f8973080cf0698886764ce38982661cc"
+  license :public_domain
+  head "https://github.com/mattrighetti/envelope.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
```

Comment:
the default livecheck should do the job (we tend to not include the latest check as it is expensive to run)
```suggestion
```

Reply:
That makes sense, I'll remove this